dailalib/api/litellm/litellm_api.py

```diff
@@ -202,6 +211,17 @@ def ask_api_key(self, *args, **kwargs):
             api_key = api_key_or_path
         self.api_key = api_key
 
+    def ask_custom_endpoint(self, *args, **kwargs):
+        custom_endpoint = self._dec_interface.gui_ask_for_string("Enter your custom OpenAI endpoint:", title="DAILA")
```

Comment:
Just add an early bail out that does not set the custom endpoint if their was no selection

Reply:
Hope the last commit is the change that you wanted